app/hooks/userUser.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext, useContext } from 'react';
+
+export interface User {
+  creatorAccount: Record<string, unknown> | null;
```

Comment:
not needed for this to merge, but we should define actual types for each of these. @KitanGarcia may have already started that, so we can try to stitch that together here

Reply:
Sure!